scripts/config/mt-gateway.ini

```diff
@@ -0,0 +1,63 @@
+
+#http service address
+addr = :5059
+
+#graphite api address
+graphite-url = http://localhost:8080
+
+#mt-whisper-importer-writer address
+#importer-url =
+
+#metrictank address
+metrictank-url = http://localhost:6060
+
+
+#default org ID to send to downstream services if none is provided (ignored if value = -1)
+default-org-id = -1
+
+
+###
+### The properties below are for configuring ingestion to kafka via the `/metrics` endpoint
+###
+
+#enable metric publishing
+metrics-publish = false
+
+#path to carbon storage-schemas.conf
+schemas-file = /etc/metrictank/storage-schemas.conf
+
+#kafka tcp address(es) for metrics, in csv host[:port] format
+kafka-tcp-addr = localhost:9092
+
+#Kafka version in semver format. All brokers must be this version or newer.
+kafka-version = 0.10.0.0
+
+#The best-effort frequency of flushes to kafka
+metrics-flush-freq = 50ms
+
+#compression: none|gzip|snappy
+metrics-kafka-comp = snappy
+
+#The maximum number of messages the producer will send in a single request
+metrics-max-messages = 5000
+
+#method used for partitioning metrics. (byOrg|bySeries|bySeriesWithTags|bySeriesWithTagsFnv) (may be given multiple times, once per topic, as a comma-separated list)
+metrics-partition-scheme = bySeries
+
+#topic for metrics (may be given multiple times as a comma-separated list)
+metrics-topic = mdm
+
+#restrict publishing data belonging to org id; 0 means no restriction (may be given multiple times, once per topic, as a comma-separated list)
+only-org-id = 0
+
+#discard data points starting with one of the given prefixes separated by | (may be given multiple times, once per topic, as a comma-separated list)
+#discard-prefixes =
+
+#enable optimized MetricPoint payload
+v2 = true
+
+#interval after which we always resend a full MetricData
+v2-clear-interval duration = 1h0m0s
```

Comment:
is this missing a `-`

Reply:
I'm actually not sure what happened there: 6bb9d11b1ba7cf8bdce09ee4fd5a61d41786f2b7